src/setupJest.ts

```diff
@@ -1,7 +1,23 @@
 import Enzyme from "enzyme"
 import Adapter from "enzyme-adapter-react-16"
 import expect from "expect"
+import { JSDOM } from "jsdom"
 
+function copyProps(src, target) {
+  Object.defineProperties(target, {
+    ...Object.getOwnPropertyDescriptors(src),
+    ...Object.getOwnPropertyDescriptors(target),
+  })
+}
+
+global.window = new JSDOM("<!doctype html><html><body></body></html>").window
+global.document = window.document
```

Comment:
This was copied over from the approach seen here: https://github.com/airbnb/enzyme/issues/1436#issuecomment-429374585

This is preferable (imo) to the alternate approach seen further up that thread (https://github.com/airbnb/enzyme/issues/1436#issuecomment-403291666) and posted in slack of `react-native-mock-render` as it requires no component mocking. Still, this approach is non-ideal as RN is not the DOM, but this is enough to get Enzyme to work for the time being. Ideally Enzyme would support RN first-class with an RN adapter, which https://github.com/airbnb/enzyme/issues/1436 tracks discussion of.